.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'bug: '
+labels: bug
+assignees: ''
+
+---
+
+**Note: For support questions, please use the [Stoplight Community forum](https://community.stoplight.io)**. This repository's issues are reserved for feature requests and bug reports. If you are unsure if you are experiencing a bug, the [Community forum](https://community.stoplight.io) is a great place to start.
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. When I run this command: 'prism mock http://gist.github.com/an-esxample-spec-file.yaml --foo --bar`
```

Comment:
@marbemac I was implying it here but should I call it out way more explicitly? Suggestion welcome. I can also fix the typo.

Reply:
Ok I just made it more better.